packages/cli/openapi-ir-to-fern/src/__test__/__snapshots__/apiture.test.ts.snap

```diff
@@ -1488,7 +1488,6 @@ Note: This operation requires an identity challenge if the financial institution
                         "memo": "school expenses",
                         "processedAt": "2024-01-15T09:30:00Z",
                         "schedule": {
-                          "amountLimit": "3456.78",
```

Comment:
It's interesting to see that this previously existed here before this change. The example is not present in the original declaration:

```yaml
- id: d62c0701-0d74
  amount: "100.00"
  state: scheduled
  memo: school expenses
  ...
  schedule:
    scheduledOn: "2023-08-01"
    frequency: monthly
    recurreneType: fixed
    count: 9
    creditsOn: "2023-08-01"
    debitsOn: "2026-08-01"
```

The `monetaryValue` type (which is what `amountLimit` is) does have the `3456.78` example associated with it, so the OpenAPI -> Fern converter should still fill in the gap (like it used to), regardless of the `ExampleTypeFactory` change.

I'll need to take a closer look at why the change affected this, but including this comment in the meantime in case anything is obvious (cc @dsinghvi)